### PR TITLE
feat: remove quorum for evm

### PR DESCRIPTION
**Pull request by j0hnfl0w**
Closes: https://github.com/snapshot-labs/sx-ui/issues/468

### CODE REVIEW

File: src/components/Block/SpaceFormVoting.vue [hunk 1]
```diff
@@ -43,12 +46,14 @@ const definition = {
       title: 'Proposal threshold',
       examples: ['1']
     },
-    quorum: {
-      type: 'string',
-      format: 'uint256',
-      title: 'Quorum',
-      examples: ['1']
-    }
+    ...(!evmNetworks.includes(props.selectedNetworkId) && {
```
Comment (Sekhmet) [inline src/components/Block/SpaceFormVoting.vue:49]: We need to change `required` too.

**Review by Sekhmet — CHANGES_REQUESTED** [review]
image

It's impossible to process in this form:
image

**Comment by j0hnfl0w** [comment]
hm, all of it dynamic quorum changes looks ugly, maybe there is a better way. Maybe it can be only temporary solution.

**Comment by Sekhmet** [comment]
> hm, all of it dynamic quorum changes looks ugly, maybe there is a better way. Maybe it can be only temporary solution.

Maybe we can avoid those ugly changes by enabling additionalProperties on the definition so we can just pass quorum in without worrying about mapping types/data.

**Comment by j0hnfl0w** [comment]
yep, here will be the same solution

**Comment by j0hnfl0w** [comment]
@Sekhmet looked again and decided to leave this dynamic staff, we still need to hide quorum from definition

**Review by Sekhmet — CHANGES_REQUESTED** [review]

File: src/views/Create.vue [hunk 2]
```diff
@@ -81,7 +81,9 @@ const settingsForm: SpaceSettings = reactive(
     minVotingDuration: 0,
     maxVotingDuration: 86400,
     proposalThreshold: '1',
-    quorum: '1'
+    ...(!evmNetworks.includes(selectedNetworkId.value) && {
+      quorum: '1'
+    })
```
Comment (Sekhmet) [inline src/views/Create.vue:86]: This doesn't really work anyway (https://github.com/snapshot-labs/sx-ui/pull/474/files#r1133918323), so let's revert it.

File: src/views/Create.vue [hunk 3]
```diff
@@ -4,7 +4,7 @@ import { useRouter } from 'vue-router';
 import { useActions } from '@/composables/useActions';
 import { useWeb3 } from '@/composables/useWeb3';
 import { clone } from '@/helpers/utils';
-import { getNetwork } from '@/networks';
+import { getNetwork, evmNetworks } from '@/networks';
```
Comment (Sekhmet) [inline src/views/Create.vue:7]: Dead import.

**Review by Sekhmet — APPROVED** [review]
tACK

**Pull request merged**